src/apps/vpn/ui/screens/home/controller/ConnectionStability.qml

```diff
@@ -22,6 +22,19 @@ Item {
         grid.flow = grid.children[0].width > (window.width - MZTheme.theme.windowMargin * 2) ? Grid.TopToBottom : Grid.LeftToRight;
         col.handleMultilineText();
     }
+
+    function handleConnectionHealthStateChange() {
+        // If the stability item is not visible, there are no connection health problems to be be processed
+        if (!visible) {
+            return;
+        }
+
+        // Notify accessibility client of connection health problems
+        let notificationText = stabilityLabel.text + " " + stabilityLabelInstruction.text;
+        console.log("MZAccessibleNotification " + notificationText);
```

Comment:
If leaving the log messages in feels necessary, would it make more sense to log it in `AccessibleNotification::Notify`?

Reply:
Good catch! Fixed.